client/jetpack-cloud/sections/sidebar-navigation/manage-selected-site.tsx

```diff
@@ -118,6 +122,15 @@ const useMenuItems = ( {
 					enabled: isAdmin && ! isWPCOM && ! isWPForTeamsSite,
 					isSelected: itemLinkMatches( path, scanPath( siteSlug ) ),
 				},
+				{
+					icon: shield,
+					path: '/',
+					link: `${ JETPACK_CLOUD_SCAN_HISTORY_LINK }/${ siteSlug }`,
+					title: translate( 'Scan' ),
+					trackEventName: 'calypso_jetpack_sidebar_scan_clicked',
```

Comment:
or perhaps use a different track event here to make it more obvious this is for scan_history

```suggestion
					trackEventName: 'calypso_jetpack_sidebar_scan_history_clicked',
```

Reply:
Thank you that make sense